importer/api/__init__.py

```diff
@@ -144,6 +142,16 @@ def import_children(course, element, parent):
         content_xml=etree.tostring(element),
         mpath=mpath,
     )
+    if element.tag == "video":
+        subname = get_video_sub(element)
+        if subname != "":
+            assets = StaticAsset.objects.filter(
+                course__id=resource.course_id,
+                path=subname,
+            )
+            for asset in assets:
```

Comment:
why would there be multiple assets with one filename and course?  That doesn't seem possible